File: ccc2011s1.py
n=int(input())
a,b=0,0
s=""
for i in range(n):
    s+=str(input())
if (s.count('s')+s.count('S'))<=(s.count('t')+s.count('T')):
    print("English")
else:
    print("French")

File: ccc2011s1.cpp
#include <iostream>
#include <string>

using namespace std;

#define MAX 100
string str;
int n,s,t;
int main ()
{
    cin >> n;
    n++;
    while (n--){
    getline(cin,str);
        for (char c:str)
        s+=(c=='s'||c=='S')?1:0,t+=(c=='t'||c=='T');
    }
   cout << ((t>s)?"English":"French") << endl;
}
